docs/LwM2M_Client/OpenThread/Adding_device.md

```diff
@@ -0,0 +1,145 @@
+# Adding nRF board to OpenThread
+
+Integrate your Nordic nRF52840-DK board to an active OpenThread Border Router.
+
+## Prerequisites
+
+- An active **OpenThread Border Router**.
+- The **nRF52840-DK** board with a USB cable.
+- Installed **minicom** or **RealTerm** (for Linux or Mac) or **PuTTy** (for Windows) or other serial communication program.
+- Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
+- A user with access to the [**Coiote IoT Device Management**](https://eu.iot.avsystem.cloud/ui/device/inventory) platform and appropriate permissions.
+
+## Connecting to the OpenThread
+
+Check your active Border Router **OTBR_IP_address** and **OTBR_port** on which your Border Router works. This will be necessary for further steps.
+
+### Connection through Wi-Fi
+
+You have to be connected to the same Wifi network as your OpenThread Border Router. To connect your device to Border Router through Wifi, you should go to `OTBR_IP_address`.
+
+!!! Note
+    There is an option to connect to your Border Router through ssh by `ssh -L OTBR_port:OTBR_IP_addres:80 -L 8081:OTBR_hostname` command, then go to: `http://OTBR_IP_address:80`. The additional forwarding for port 8081 is necessary for the ***Topology*** view to work. Also, port 80 is necessary due to the HTTP protocol.
+
+When you are connected you should see the OpenThread main page:
+
+![OT Main Page](images/ot_main_page.PNG "OT Main Page")
+
+### Creating an OpenThread network
+
+On the left side, select the option ***Form***, and a new page will be displayed for the network creation. Click the form and a message should pop up to let you know about the operation's success. ![Form Network](images/form_network.png "Form Network"){:style="float: left;margin-right: 1177px;margin-top: 17px; margin-bottom: 17px;"}
+
+On the left-side menu, click ***Topology*** to see the role of Border Router ![Border Router topology](images/border1.png "Border Router topology"){:style="float: left;margin-right: 1177px;margin-top: 17px; margin-bottom: 17px;"}
+
+### Starting Commission
+
+To start the commissioning process click ***Commission*** on the left-side menu and write down Joiner PSKd password.
+
+!!! Important
+    The PSKd needs to conform to the following: Length: 6-32 characters, encoding: base32-thread (0-9, A-Y excluding I, O, Q, and Z for readability).
```

Comment:
OK this page that you link https://community.silabs.com/s/article/openthread-faq?language=en_US does refers to this as a base32-thread encoding. But this one does not: https://openthread.io/guides/border-router/external-commissioning/prepare?hl=en (sating just that it should be 0-9, A-Y excluding I, O, Q, and Z for readability). I did tried to look up what base32-thread encoding means and found https://github.com/openthread/openthread/issues/2156 where someone asks if there is a format reference for this. And also a link to a different link: https://github.com/openthread/openthread/pull/2185 that I didn't read through. Maybe if we wand to use the phrase base32-thread that we should find the official reference and link that.